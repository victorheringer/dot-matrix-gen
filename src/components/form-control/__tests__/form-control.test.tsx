import { render } from "@testing-library/react";
import { FormControl } from "../index";

describe("FormControl", () => {
  it("should render correctly", () => {
    const tree = render(<FormControl />).asFragment();
    expect(tree).toMatchSnapshot();
  });
});
