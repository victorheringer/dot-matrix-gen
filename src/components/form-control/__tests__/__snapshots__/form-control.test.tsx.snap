// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormControl should render correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-bczRLJ UOxcP"
  />
</DocumentFragment>
`;
